modules/phase_field/tests/SoretDiffusion/tests

```diff
@@ -0,0 +1,22 @@
+[Tests]
+  [./SoretDiffusion_split_test]
```

Comment:
I suggest ```soretdiffusion_split_test``` -> ```split``` etc.

otherwise the test harness will report
```
SoretDiffusion.SoretDiffusion_split_test......
```
Class name is duplicated and we know that it its a _test_.